examples/jsm/libs/flow.module.js

```diff
@@ -1452,7 +1452,7 @@ class Node extends Serializer {
 
 	serialize( data ) {
 
-		const { x, y, style } = this.getPosition();
```

Comment:
@sunag The object returned by `getPosition()` does not hold a `style` property. 

It seems more correct to serialize the value of `this.style`.